mix/ClientModel.cpp

```diff
@@ -181,7 +188,7 @@ void ClientModel::setupState(QVariantMap _state)
 				transactionSettings.parameterValues.push_back(param);
 			}
 
-			if (transaction.value("executeConstructor").toBool())
+			if (contractId == functionId || functionId == "Constructor")
```

Comment:
The "Constructor" label is no more. This code is to support old project files, will be removed once everyone upgrades